### DISCUSSION TITLE
Add to Frigate+ button disappeared

### DISCUSSION THREAD

**Discussion by blacknell** [blakeblackshear/frigate#14881] [Ask A Question] [answered]
Been verifying labelling my images in plus, and then went back to Frigate to flag some more candidates. All in advance of training a new model on the latest.

The add to Frigate+ button seems to have disappeared (I admit been a while since I used it.

Snapshots is on (I just checked in MQTT) but can't find the icon/button in GUI to check/change

Any ideas?

I'd like to verify a few more before training as theyve got waste_bins and a few couriers I would like hoping take advantage of

**Comment by hawkeye217** [comment]
What version of Frigate are you using?

**Comment by blacknell** [comment]
0.14.1

**Comment by hawkeye217** [comment]
You'll want to use the Frigate+ panel accessible from the sidebar, then. You can also submit frames from History view by using the Frigate+ button in the video controls. 

If you don't see these options available in your installation, you will need to configure your Frigate+ API key environment variable according to the Frigate+ setup docs.

**Comment by blacknell** [comment]
Yeah, that's what I'm looking at.

<img width="618" alt="image" src="https://github.com/user-attachments/assets/9eee8df5-29b3-4701-bdc1-a156abb50a16">

Just can't see the submit to frigate+ UI for the individual snapshots

**Reply by NickM-27** [reply]
If you click on a snapshot it gives you the option to submit

**Comment by blacknell** [comment]
If I click on snapshot it just navigates to a fuller screen image. I know this used to be obvious

<img width="1256" alt="image" src="https://github.com/user-attachments/assets/b7978ea4-f995-41f2-9029-cba0a17a9206">

**Reply by NickM-27** [reply, marked as answer]
you'll need to zoom out to see the buttons for submitting. This issue is fixed in the next version of frigate

**Comment by blacknell** [comment]
Ok. something odd. It seems to jump zoomed in and after **several** Cmd- and Cmd0 I could then see the buttons

**Comment by Hammerhand17** [comment]
Im on 0.15 and I still have this issue: the buttons are not there (neither in chrome, zen, firefox, edge); in android, they appear when I click on the video. I have:

```
    detect:
      width: 800
      height: 600
      fps: 5
```

The zoom "trick" doesn't work for me.

Just to clarify: in previous versions I had a button when I parked a false positive to be sent to frigate +. Now I can't see it.

**Reply by NickM-27** [reply]
> in android, they appear when I click on the video.

they won't show for a video, I think you should create a new discussion
